ui/components-react/src/components-react/filter-builder/FilterBuilderRuleGroup.tsx

```diff
@@ -206,3 +212,27 @@ const PropertyFilterBuilderGroupOrRule = React.memo(
     );
   }
 );
+
+const useRulePropertyFocus = (
+  currentGroupItemsLength: number,
+  groupElement: HTMLDivElement | null
+) => {
+  const previousGroupItemsLength = React.useRef<number>(0);
```

Comment:
```suggestion
const useRulePropertyFocus = (
  currentGroupItemsLength: number,
  groupElement: HTMLDivElement | null
): [() => void, React.Ref<HTMLDivElement>] => {
  const ref= React.useRef<HTMLDivElement>(null);
  return [() => {}, ref];
}
```

I was thinking about something like that?

Reply:
I assumed that we don't need `groupElement: HTMLDivElement | null` as a second argument, fixed